internal/testutil/mig/test_case.go

```diff
@@ -4,11 +4,13 @@ import (
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
 	"github.com/stretchr/testify/require"
 )
 
 func CreateAndRunTest(t *testing.T, test *resource.TestCase) {
 	t.Helper()
+	acc.SkipInUnitTest(t)
```

Comment:
this is needed because although ParallelTest doesn't run in unit test, but CreateTest might run but we don't want that